src/controllers/user.controller.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-console */
+/* eslint-disable no-useless-return */
+'use strict';
+
+const userService = require('../services/users.service');
+
+const get = (req, res) => {
+  const allUsers = userService.getAllUsers();
+
+  if (!allUsers) {
+    res.status(404).send('Not Found: The specified entity does not exist');
+
+    return;
```

Comment:
here for example. you don't need to return. after `res.send` the function ends its call

Reply:
fix it everywhere